custom-vu/src/main/kotlin/jces1209/vu/ScenarioSimilarities.kt

```diff
@@ -218,33 +218,44 @@ class ScenarioSimilarities(
         browseIssueTypes: Action,
         browseProjectRoles: Action
     ): List<Action> {
+        val readTrafficShapeConfig = System.getenv("readTrafficShapeConfig")
```

Comment:
wrt to the other comment, I think we don't need this variable, just try read the file, catch the exception if having issue and fallback to default